type_check_failed,error_msgs
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in intersectionWithPath: Name 'path0' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in simpleSplit: Name 'segment0' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in processes: Built-in function open was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in is_older_than: 'datetime' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in _get_options: No attribute 'add_argument' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in transform_targeting_data_to_dfp: No attribute 'day_part' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in make: Name 'mds_create' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in mon_status_check: Name 'code' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in catch_mon_errors: No attribute 'warning' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 27, in mon_status: Name 'out' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in keyring_parser: Name 'line' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in mon_destroy: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 31, in mon_create_initial: No attribute 'conn' on Set[nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 14, in make: Name 'mon_create_initial' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in repo: No attribute 'codename' on logging.Logger [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in map_graf_nodes: Name 'j' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in batch_put_proxy: Name 'objs' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in put_backup_proxy: Name 'objs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in init: Name 'other_addr' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in apply_idf_weighting: Name 'idfs' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_information_content: Name 'ics' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in __init__: No attribute 'sqlite' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in __init__: Name 'conn' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in background_thread: Name 'date' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _read_response: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in write_json_into_file: Function TextIO.write was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_network_data: Name 'service' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_line_item_creatives: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_geography_targets: No attribute '_run_pql_query' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in get_adunit_targets: No attribute '_run_pql_query' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in validate_credentials: Name '_val' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _adunit_from_dfp: Name 'include_descendants' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in transform_geography_targeting_to_dfp: No attribute 'get_includes_and_excludes' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _handle_fetch_banned: Name 'index' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in list_runnable_jobs: Name 'desc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in stop_running_job: Name 'job_info' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in pack_job_error: Name 'zip_filename' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __init__: Name 'msg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in apply_func: Name 'expr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in testMicroBlogParser: Name 'parser' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in testUserInfoParser: No attribute 'parse' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in testUserInfoParserForSite: Name 'test_url' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in testFriendParser: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in testBatchPutGet: No attribute 'assertEqual' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in add: Name 'pos' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in __contains__: No attribute 'hash' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_data_files: Name 'bash_comp_dest' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_catkin_build_with_whitespace_in_paths: Name 'cmd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in levenshtein_distance: unsupported operand type(s) for +: 'distances: range' and '1: int' [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_words: No attribute 'orth_' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_ngrams_filter: No attribute 'spacy_doc' on List[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_ngrams_min_freq: No attribute 'spacy_doc' on List[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_ngrams_good_tag: Name 'observed' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in load_env: Can't assign attribute 'cached_cmake_prefix_path' on list [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_get_anonymous_user_works: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in run_vanilla_any_poly: Name 'f' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in bench_load1: Name 'o' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in bench_load1_short: Name 'o' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: No attribute 'fields' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in is_forbidden: No attribute 'user' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_get: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_export_one_page: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_get_invalid: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _encode_key_name: unsupported operand type(s) for item assignment: bytes [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _encode_number: unsupported operand type(s) for item retrieval: 'val: float' and 'slice(None, None, None): slice' [unsupported-operands]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in open: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'shuffle_urls' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 46, in execute: No attribute 'opener' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in parse: Name 'ifp' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 7, in load_raw: Name 'ceph_conf' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _init_counter_server: No attribute 'rpc_server' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _init_budget_server: Name 'budget_dir' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _init_speed_server: Name 'speed_dir' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in shutdown: Name 'handle' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 31, in _generate_file: No attribute 'map_handles' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 29, in _destroy_file: No attribute 'legal_files' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in open_file: Name 'newfile' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in concat: Name 'val' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in process: No attribute 'handle_file' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in save: Name 't' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in init_deduper: No attribute 'is_multi_process' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in _init_function_servers: No attribute 'rpc_server' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in init_functions: No attribute 'ctx' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _configure_proxy: No attribute 'addr' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_object: No attribute 'instance' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get: No attribute 'instance' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_link_templates: Name 'uri' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in post: Name 'instance' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in _decode_prop: Name 'prop_type' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 17, in _decode_prop_obj: No attribute 'value' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in list_package_files: Function str.replace was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 27, in get_package_and_resource_name: No attribute 'startswith' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _search_for_package_and_resource: Name 'resource' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in extract_zip_resource: No attribute 'extract' on None [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in counter_key: Name 'pk' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in is_series: No attribute 'is_successor' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __iter__: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __str__: Name 'node' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _classes: No attribute 'classes' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __str__: Name 'val' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __setitem__: Name 'values' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in run: No attribute 'name' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_no_identifier: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_get: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in post: No attribute 'render' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in resume_from: Name 'position' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in create_records: Name 'peeps' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _get_all_files: 'glob' object is not callable [not-callable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in fmt_filename: Name 'pyfile' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in process_all: No attribute 'worker' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in __iter__: Name 'record' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'func' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 5, in call: No attribute 'push' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in view: Name 'items' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __str__: No attribute 'vertices' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in parse_parameters: No attribute 'split' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in random_binary: unsupported operand type(s) for +: int and str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 5, in _rand_bitstring: Name '_' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in call: Name 'line' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in is_closed: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _prune_all: Name 'start' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'MAX_SIZE' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 4, in __str__: Name 'chars' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in process: Name '_key' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __str__: Name 'k' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in make_substring: unsupported operand type(s) for item retrieval: 'k: int' and 'slice' [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'conn' on tuple [not-writable]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __init__: Can't assign attribute 'db' on Dict[str, Any] [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in _create_user: No attribute 'db' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in pred_get_valid_funcs: Name 'name' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 5, in insert_all: Name 'n' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in insert_all: unsupported operand type(s) for item retrieval: 'n: int' and ''name': str' [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in insert_all: Name 'n' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _view: Built-in function len was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in sort: No attribute 'strings' on int [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __str__: Name 'name' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in prnt: Name 'offset' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in substring_dict: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in clear_from: Name 'offset' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in flush_blocks: Name 'writer' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in _setup: Can't assign attribute '_wrapped' on str [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in raise_dep_error: Name 'dep' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_get: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __abs__: No attribute 'left' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in run: Name 'index' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: No attribute 'get_root' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __setitem__: unsupported operand type(s) for item assignment: str [unsupported-operands]
File ""dummpy_input_file"", line 6, in __setitem__: unsupported operand type(s) for item assignment: list and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in skew_to_dec: unsupported operand type(s) for +: str and float [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 76, in main: Name 'version_display_string' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in read_bin: Name 'bin_reader' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in md5: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in file_to_file: Name 'hash_digest_one' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in version_read: Name 'minor_match' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'logger' on tuple [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute 'logger' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 3, in log: Name 'msg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __contains__: Name 'svalue' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __repr__: No attribute 'end' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in step_n: Name 'n' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in find_successor: No attribute 'parent' on None [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in remove: Name 'successor' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in populate_bst: Name '_' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _disperse: Name '_' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test: Name '_digit' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in truncate: Name 'store' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in read_header: Name 'header' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in _save_header: Name 'params' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in hash_header: No attribute 'encode' on bytes [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_hex_txhash: Name 'the_hash' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in get_child_type_choices: No attribute 'append' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in _get_real_admin_by_ct: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in add_view: No attribute 'add_view' on int [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 5, in changeform_view: Name 'real_admin' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in history_view: Name 'real_admin' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in add_type_view: No attribute 'method' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in delete_confirmation_template: Name 'opts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in relevant: Name 'cvl' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __init__: Name 'params' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_coin: Name 'coin_rec' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in apply_tx: Name 'coin_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in get_new_address: Name 'index' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_new_genesis_address: Name 'index' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in update_config: No attribute 'config' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get_queryset: Name 'place' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in items: Name 'results' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in POST: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Function Thread.__init__ was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 28, in POST: unsupported operand type(s) for item retrieval: 'merkle: Iterator' and '1: int' [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 4, in get_kwargs: Name 'kwargs' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in __neg__: Name 'kwargs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __str__: Name 'field' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_block_height: Name 'req' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get_chunk: No attribute 'read' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in coloraddress_to_bitcoinaddress: Name 'colorid' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_data: Name 'raw' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in listassets: Name 'assets' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in getbalances: Name 'balances' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in newaddress: Name 'coloraddress' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in listaddresses: Name 'asset' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_TSO_result_type: Name 'e' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_get_season: Name 'season' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_recap: No attribute 'recap' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in clean: No attribute 'file' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_redirect_url: Name 'space' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in get_queryset: Name 'place' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in get_queryset: Name 'place' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in ones: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _int_to_hex: unsupported operand type(s) for -: 'str' and 'int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _get_read_size: Name 'last1' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in p2ptrade_init: Can't assign attribute 'p2p_agent' on Dict[str, int] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in edtAddressValidate: No attribute 'edtAddress' on bool [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in edtAmountValidate: No attribute 'edtAmount' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in __init__: No attribute 'wallet' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_get_new_genesis_address: Name 'addr2' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_get_all_addresses: Name 'addrs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in test_composed: Name 'same' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 16, in test_other: Name 'signed' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_configuration: No attribute 'read' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_accountlist: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_bad_operator_rejection: No attribute 'process_filters' on List[List[str]] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_multiple_filters_mix_ops: Name 'filter_string' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_multiple_dimensions: Name 't' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __unicode__: Name 'truncated_body' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in update_contacts: No attribute '__iter__' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in is_monocolor: Name 'asset' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in name: Name 'name' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in edit: Name 's' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in __init__: Can't assign attribute 'ps1' on tuple [not-writable]"
False,
True,"File ""dummpy_input_file"", line 26, in run: No attribute 'start_dirpath' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in run: Name 'the_viewer' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in from_color_ids: Name 'color_desc_list' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in menu: Name 'runs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_check_permissions: Name 'user_permissions' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_no_profile: Name 'user' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_change_email_form: No attribute 'is_valid' on Dict[str, Union[Dict[str, str], Tuple[str, List[str]]]] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in test_TSO_count: No attribute 'assertEqual' on int [attribute-error]
File ""dummpy_input_file"", line 16, in test_TSO_count: No attribute 'assertEqual' on float [attribute-error]
File ""dummpy_input_file"", line 16, in test_TSO_count: No attribute 'assertEqual' on str [attribute-error]
File ""dummpy_input_file"", line 16, in test_TSO_count: No attribute 'assertEqual' on List[nothing] [attribute-error]
File ""dummpy_input_file"", line 16, in test_TSO_count: No attribute 'assertEqual' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_TSO_inclEntities: No attribute 'set_include_entities' on List[bool] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_TSO_filters: No attribute 'assertTrue' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_TO_exceptions: No attribute 'set_search_url' on Type[NotImplementedError] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_TS_search_usertimeline_iterable: unsupported operand type(s) for item retrieval: 'pages: int' and '0: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_TS_search_tweets: unsupported operand type(s) for item retrieval: 'todo: bool' and ''content': str' [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_TS_methods_exceptions: Name 'ts' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_TSE_common: Built-in function len was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in __init__: No attribute 'type' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in weighted_random_baseline: Name 'n' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in _communicate: Name 'stdout' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in save: Name 'p' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in extend: No attribute 'name' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in end_tag: Name 'tag' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in all: Name 'elem' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 5, in clear: Function dict.clear was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __init__: unsupported operand type(s) for item assignment: range [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in f: Name 'op' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _get_option: Name 'root' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in render: Name 'child' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in show_group: Name 'child' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 66, in render: Name 'group' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 67, in render: Name 'child' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in _train: No attribute 'extension' on Dict[Any, int] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in is_short_syllable: Name 'i' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in R1: Name 'm' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in vowel_consonant_pairs: Function enumerate.__init__ was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in serve: Name 'cov' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _multiple: Name 'm' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_fillna: Name 'expr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_respect_set_limit: Name 'table' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_disable_query_limit: Name 'table' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_column: Name 'ref' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in has_reduction: Name 'arg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in close: Name 'k' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _close_cursor: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_interactive_non_compilable_repr_not_fail: Name 'table' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in all_equal: Name 'a' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in __init__: No attribute '_validate_args' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in output_type: Name 'klass' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in distinct_roots: No attribute 'extend' on tuple [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _can_cast_implicit: Name 'rule' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in find_base_table: Name 'r' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in distance: Invalid keyword argument method to function z__pytype_partial__unknown593.distance [wrong-keyword-args]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in load: No attribute 'splitlines' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in readline: No attribute 'bytes_read' on list [attribute-error]
File ""dummpy_input_file"", line 18, in readline: Can't assign attribute 'bytes_read' on list [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __exit__: Name 'val' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _get_deprecated_option: Name 'd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in config_prefix: Name 'register_option' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 6, in is_instance_factory: Name 'type_repr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in get_result: Name 'subbed_node' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in _sub_arg: Name 'subbed_arg' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _sub: Name 'helper' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 34, in vector: No attribute 'terms' on None [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in copy: Name 'd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in classify: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in _tree: Name 'type' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in _search: No attribute 'children' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_sign: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_integer_promotions: Name 'expr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _value_list: Name 'formatted' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in condensed_nearest_neighbor: No attribute 'vector' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in predict: No attribute '_classifier' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in concepts: No attribute '_terms' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _eintr_retry_call: 'tuple' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 37, in _communicate_with_select: No attribute 'stdin' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in unquote_plus: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in cleanup: No attribute 'key' on float [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in poll: Name 'w' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 17, in get_tokens_unprocessed: No attribute 'group' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in validate: Name 'value_valid' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in collect: Name 'arg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_binary_infix_operators: Name 'cases' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_binary_infix_parenthesization: Name 'cases' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_isnull_notnull: Name 'cases' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_casts: Name 'cases' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_negate: Name 'cases' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_dtype_float32: Name 'inferred' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_dtype_uint32: Name 'inferred' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_dtype_uint64: Name 'inferred' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_dtype_datetime64: Name 'inferred' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_dtype_timedelta64: Name 'inferred' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_create_table_from_dataframe: No attribute 'alltypes' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in lift: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _number_literal_format: Name 'value' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _timestamp_literal_format: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_timestamp_extract_field: Name 'cases' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_timestamp_literals: Name 'cases' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_any_all: Name 't' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _is_null: Name 'arg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _searched_case: Name 'op' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in table: Name 'node' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in clone: Name 'kwargs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in validate_match: Name 'regex_pattern' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_age: Name 'diff' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in _case_sort_by: Name 'table' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_parse_url: No attribute '_check_expr_cases' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_repeat: Name 'cases' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_translate: No attribute '_check_expr_cases' on List[Tuple[Any, str]] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_lpad: No attribute '_check_expr_cases' on List[Tuple[Any, str]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_rpad: Name 'cases' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_find_in_set: Name 'cases' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in formatter: Name 'right_arg' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in translator: Name 'sa_arg' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_compile_verify: Name 'supported_expr' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in setUp: Can't assign attribute 'mm' on str [not-writable]"
False,
True,"File ""dummpy_input_file"", line 5, in tearDown: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_task_end_to_end: Name 'msg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in tearDown: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __contains__: Name 'other' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_numeric_unary_builtins: Cannot unpack 14 values into 2 variables [bad-unpacking]"
False,
True,"File ""dummpy_input_file"", line 3, in test_log_other_bases: Name 'cases' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_round: Name 'cases' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_bucket_assign_labels: unsupported operand type(s) for item retrieval: List[int] and str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 3, in test_field_in_literals: No attribute '_check_expr_cases' on List[Tuple[Any, str]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_literal_in_list: Name 'cases' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_analytic_exprs: Name 'cases' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_substr: No attribute '_check_expr_cases' on List[Tuple[Any, str]] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_like: No attribute '_check_expr_cases' on List[Tuple[Any, str]] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_re_extract: Name 'cases' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_re_replace: Name 'cases' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _compile_table_set: Name 'helper' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _add_groupby: Name 'having_args' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _add_order_by: Name 'sort_expr' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _add_limit: Name 'n' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_tpch_self_join_failure: Name 'year' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute '_iterations' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in __init__: Can't assign attribute '_svm' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_unpartitioned_table_get_schema: No attribute 'con' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in test_insert_select_partitioned_table: Name 'year' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in test_add_drop_partition: No attribute 'add_partition' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in apply_filter: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _walk: Name 'is_valid' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in _validate_projection: No attribute 'name' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in format_subqueries: No attribute 'write' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in next: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in run: Name 'status' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _resize_node: Name 'params' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in connect: Name 'conninfo' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in run: Name 'stdin' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_meta_dict: No attribute 'text' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in format_table_set: Name 'helper' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in format_where: Name 'fmt_preds' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 14, in format_postamble: No attribute 'write' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in _check_begin_patch: Name 'mod' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in _cleanup_patch: Name 'fileset' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _do_SET_PATH: Name 'path' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_get_meta_data: Name 'meta_data' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_list_container_objects: Name 'objects' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_get_container: Name 'container' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_create_container_invalid_name: Name 'container' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in download_object_success_not_found: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_destroy_node: Name 'ret' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _to_image: Name 'n' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in list_nodes: Name 'groups' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in list_sizes: Name 'include_cluser_instances' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in ex_list_availability_zones: No attribute 'path' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in ex_describe_tags: No attribute '_findtext' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in ex_describe_addresses_for_node: Name 'node_elastic_ips' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in create_node: No attribute 'id' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in destroy_node: Name 'res' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute 'path' on str [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_string_join: No attribute '_check_expr_cases' on List[Tuple[Any, str]] [attribute-error]"
False,
